crates/conjure_core/src/rules/base.rs

```diff
@@ -454,6 +454,50 @@ fn min_to_var(expr: &Expr, mdl: &Model) -> ApplicationResult {
     }
 }
 
+/**
+ * Turn a Min into a new variable and post a global constraint to ensure the new variable is the minimum.
```

Comment:
should say maximum instead of minimum. also here and in the min rule we shouldn't say "global constraint". I understand why you use the term, but the same term is used to describe something else in CP. call them top level constraints?

Reply:
Changed the comments for both min and max